Undefined names in test_product.py

https://github.com/mirumee/saleor/blob/master/tests/dashboard/test_product.py#L268-L273

flake8 testing of https://github.com/mirumee/saleor on Python 3.6.3

$ __flake8 . --count --select=E901,E999,F821,F822,F823 --show-source --statistics__
```
./tests/dashboard/test_product.py:268:12: F821 undefined name 'response'
    assert response.status_code == 302
           ^
./tests/dashboard/test_product.py:269:34: F821 undefined name 'response'
    assert get_redirect_location(response) == reverse('dashboard:product-list')
                                 ^
./tests/dashboard/test_product.py:271:14: F821 undefined name 'product_list'
    for p in product_list:
             ^
3     F821 undefined name 'response'
3
```

--- a/tests/dashboard/test_product.py
+++ b/tests/dashboard/test_product.py
@@ -252,6 +252,13 @@ def test_view_product_bulk_update_unpublish(admin_client, product_list):
 
     response = admin_client.post(url, data)
 
+    assert response.status_code == 302
+    assert get_redirect_location(response) == reverse('dashboard:product-list')
+
+    for p in product_list:
+        p.refresh_from_db()
+        assert not p.is_published
+
 
 def test_product_variant_form(product):
     variant = product.variants.first()
@@ -264,13 +271,6 @@ def test_product_variant_form(product):
     form.save()
     variant.refresh_from_db()
     assert variant.name == example_size
-
-    assert response.status_code == 302
-    assert get_redirect_location(response) == reverse('dashboard:product-list')
-
-    for p in product_list:
-        p.refresh_from_db()
-        assert not p.is_published
 
 
 def test_view_ajax_products_list(admin_client, product):

